app/src/main/java/co/condorlabs/customcomponents/customspinner/readme.md

```diff
@@ -31,7 +31,8 @@ Create a new instance in your layout
 |  Unit | *`fun setIsRequired(required: Boolean)`* <br> make the field required|
 |  Unit | *`fun setData(data: List<SpinnerData>)`* <br> set the spinner data|
 |  Unit | *`fun setItemSelectedById(id: String)`* <br> Select an item by his id|
-|  Unit | *`fun setItemSelectedHint()`* <br> Select the hint item|
+|  Unit | *`fun clearField()`* <br> Clear the actual selection|
```

Comment:
Is this function missing?
### isHintPosition(position: Int)

Reply:
that is and adapter function and its not expose